core/src/types.ts

```diff
@@ -63,6 +63,8 @@ export declare interface CollateralRow extends CollateralConfig, Partial<MakerPa
     marketUnitPrice?: BigNumber | string;
     tokenAddress?: string;
     tokenAddressError?: string;
+    autoRouteQuote?: BigNumber;
+    autoRouteExchanges?: string | undefined[];
```

Comment:
Is this what you meant? `?` already specifies that
```suggestion
    autoRouteExchanges?: string[];
```